erpnext/agriculture/doctype/harvest/harvest.py

```diff
@@ -3,8 +3,62 @@
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
-# import frappe
+import frappe
+import json
 from frappe.model.document import Document
 
 class Harvest(Document):
 	pass
+
+@frappe.whitelist()
+def create_stock_entry(harvest):
+	harvest = frappe.get_doc(json.loads(harvest))
+	if stock_entry_exists(harvest.get('name')):
+		return frappe.msgprint(_('Stock Entry has been already created against this harvest'))
+
+	stock_entry = frappe.new_doc('Stock Entry')
+	stock_entry.harvest = harvest.get('name')
+	stock_entry.purpose = 'Material Receipt',
```

Comment:
Would this be a Manufacture instead? @neilLasrado, what do you think?

Reply:
Added purpose as material Receipt. consulted with neil